lib/errors.js

```diff
@@ -176,5 +176,10 @@ module.exports = {
         this.name = 'BAD_TIMESTAMP';
         this.message = 'Invalid ISO8601 timestamp [' + payload + ']';
```

Comment:
CHANGES_NEXT_RELEASE entry should be added

(Maybe at the end we will group all the NGSI-LD ones together in a single entry, but by the moment let's add the one corresponding to each PR)

Reply:
On the other hand, I understand this PR doesn't need any addition/modification to the documentation, does it?

Reply:
Maybe the following behaviour should be documented (not fully sure)?

> The NGSI v2 `TimeInstant` element has been mapped onto the NGSI-LD `observedAt` property
>
> The NGSI v2 `metadata.unitCode` attribute has been mapped onto the NGSI-LD `unitCode` property

Reply:
CNR has been updated:  1da050b

Reply:
What about documentation modifications?

Reply:
Minimal documentation has been added: fa90072